### PR TITLE
Dev: Fixed livermask + updated README docs + added loader

### LINKED ISSUE #13: Refactor: remove redundancies
It is great that the `main` branch is as clean as possible for current and future devs. Hence, removing environmental directories such as `.vs/` and `.idea/` is of interest.

### LINKED ISSUE #14: Setup README for development
The current README was taken from the [Flask](https://pypi.org/project/Flask/) project. We should modify to be more suitable for new developers, to have a better suited tutorial on how to setup the project for development.

### PR DESCRIPTION
**Pull request by andreped**
## Changes:
- Modified README and tested that it works on macOS
- Added `livermask` and `flask` as dependencies
- Renamed name in `setup.cfg` to `SINTEFMIACatalogue`
- Minor refactoring
- Updated port to match current setup
- Set correct livermask JS script in jinja2 template
- Added loader custom CSS to livermask
- Fixed livermask processing (upload from frontend, runs in backend, and result is auto-downloaded when finished - work in progress, but at least it works)

### CODE REVIEW

File: templates/livermask.jinja2 [hunk 1]
```diff
@@ -4,16 +4,20 @@
     <div class="my-5 pt-5 container">
       <div class="row justify-content-center align-content-center">
         <div class="col-sm-6">
-          <form action="http://localhost:88/process" method="post" enctype="multipart/form-data">
+          <form action="http://localhost:5000/process" method="post" enctype="multipart/form-data">
```
Comment (jpdefrutos) [inline templates/livermask.jinja2]:
```suggestion
          <form action="{{ url_for('process') }}" method="post" enctype="multipart/form-data">
```

File: templates/processing.jinja2 [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% extends "application_template.jinja2" %}
+
+{% block app_content %}
+    <div class="my-5 pt-5 container">
+      <div class="row justify-content-center align-content-center">
+        <div class="col-sm-6">
+          <form action="http://localhost:5000/process" method="post" enctype="multipart/form-data">
```
Comment (jpdefrutos) [inline templates/processing.jinja2]:
```suggestion
          <form action="{{ url_for('process') }}" method="post" enctype="multipart/form-data">
```

File: main.py [hunk 3]
```diff
@@ -1,27 +1,73 @@
...
 def application(app_name):
     return render_template("{}.jinja2".format(app_name.lower()), **{'app_name': app_name.capitalize()})
 
 
-# @app.route("process/<string:app_name>", methods=['GET', 'POST'])
-# def process(app_name):
-#     if request.method == 'POST':
-#         # Forward the uploaded data to the corresponding service
-#         APP_DICT[app_name].
+@app.route("/")
```
Comment (jpdefrutos) [inline main.py:27]: You can use the same route i.e., "/", for multiple functions depending on the communication method used, e.g., GET (default), POST, etc.
Linking the same route with the same method, to different functions is not advised.

I would suggest merging the functions "application" and "process", and use the same template (or collection of templates) for all the services . These can be adapted, and changes e.g., show/hide the spinning circle, based on the status of the task.

File: templates/processing.jinja2 [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+                </div>
+                <div class="row justify-content-center">
+                    <div class="col-6">
+                        <button type="reset" class="btn btn-secondary" style="width: 100%">Clear</button>
+                    </div>
+                    <div class="col-6">
+                        <button type="submit" class="btn btn-primary" style="width: 100%">Submit</button>
+                    </div>
+                </div>
+                <div class="row justify-content-center">
```
Comment (jpdefrutos) [inline templates/processing.jinja2]:
```suggestion
                <div class="row mt-5">
```

File: templates/processing.jinja2 [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+                    <div class="col">
```
Comment (jpdefrutos) [inline templates/processing.jinja2]:
```suggestion
                    <div class="col justify-content-center d-flex">
```

**Review by jpdefrutos — CHANGES_REQUESTED** [review]
Good job! minor suggestions and comments which should be addressed before merging

File: main.py (hunk 3 above)
Comment (andreped) [reply main.py:27]: Yh, I agree. I was just playing around with stuff here.

**Comment by andreped** [comment]
They have been corrected for now, @jpdefrutos. LGTM. I did not have any other comments, hence, merging PR.

**Pull request merged**